### ISSUE TITLE
TRIQS installation (openSUSE in Virtual Box)

### ISSUE THREAD

**Issue by phystd** [TRIQS/packaging#1]
Hello-,

I am having the following issues related to TRIQS installation - 
(a) I am trying to install TRIQS in Ubuntu (installed in a virtual box). When I use the Ubuntu Debian package commands listed on the website, I get messages that there is no OpenGPG data found and a 404 Not found  [IP 104.26.7.230 443]. I have internet connectivity from my virutal box. 
(b) Is it possible to install TRIQS in openSuse? My installation attempt kept on giving me a MPI_C and MPI_CC not found error? I have the mpi and mpich packages installed. 

I would appreciate any help that you can offer. Thanks.

**Comment by Wentzell** [comment]
Dear @phystd,

> I get messages that there is no OpenGPG data found and a 404 Not found [IP 104.26.7.230 443]

This sounds indeed like a problem with your internet connection.
Are you able to install other packages through the apt package manager?

> Is it possible to install TRIQS in openSuse? My installation attempt kept on giving me a MPI_C and MPI_CC not found error? I have the mpi and mpich packages installed.

We have unfortunately no experience with building TRIQS on openSuse, but it should be possible.
You would have to make sure that all [prerequisites](https://triqs.github.io/triqs/latest/installation/requirements.html#requirements) are installed before you proceed with the [build process](https://triqs.github.io/triqs/latest/install.html#installation-steps).

Alternatively you could use our [TRIQS Anaconda Package](https://github.com/conda-forge/triqs-feedstock)  which should work independently of the linux distribution.

**Comment by phystd** [comment]
Thanks. Does the TRIQS package install the dependencies automatically, or do I need to ensure that those are in place first before executing the Anaconda package commands? Also, will this install only the python based codes for future applications or all the possible supported TRIQS development languages?

**Comment by phystd** [comment]
Following up on my previous issue, I cannot find an Anaconda package to install conda. Is there a work around?

**Comment by phystd** [comment]
A. I was able to install TRIQS using the Anaconda Package in my OpenSuse LINUX. However, I cannot figure out where it is installed, since there is no triqs folder. Any suggestions on how to find this? I also had to create a new environment called myenv to proceed with installation. Are the files located in this myenv? When I do ipytriqs test.py, i get command not found. How can I fix this?

B. I want to install -CTHYB. Is there a Anaconda package installation available for this? If not, do I need to build this for OpenSuse?

Thanks.

**Comment by Wentzell** [comment]
Dear @phystd,

Both the C++ and the Python parts of triqs will be installed into the folder structure of anaconda,
i.e. `path_to_conda/include/`, `path_to_conda/lib`, ...

Once you have [activated](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment) your conda environment you should be able to import the
triqs python modules as can be easily checked  with

```bash
python -c "from pytriqs import version; print version.version"
```

> B. I want to install -CTHYB. Is there a Anaconda package installation available for this? If not, do I need to build this for OpenSuse?

We have not yet created anaconda packages for the triqs applications unfortunately.
You can however compile cthyb against the triqs version installed in your anaconda path.
Be sure to activate your conda environment, load the triqs environment variables with

```bash
source path_to_conda/share/triqsvars.sh
```

and follow the compilation steps for cthyb outlined [here](https://triqs.github.io/cthyb/latest/install.html#installation-steps).

**Comment by phystd** [comment]
I was able to run the tutorial example codes, but not using the ipytriqs or pytriqs commands. I did the following (i) After changing the environment (as suggested), I simply did "python myfile.py". (2) I had to introduce a plt.show() command in the python code to produce the local Green's function plot. Otherwise it does not show. (3) The pytriqs.gf.local is deprecated. Only pytriqs.gf should be used. 

Questions:
A. When run the tutorial code -
https://triqs.github.io/triqs/1.4/tutorials/gfs/lattice_gfs_tutorial_python.html

it says multivar cannot be found. How can I fix this?

B. When I execute 
cmake -DMeasureG2=OFF ../cthyb.src at the prompt 
I get 
cmake: symbol lookup error: /usr/lib64/libssh.so.4: undefined symbol: EVP_KDF_CTX_new_id, version OPENSSL_1_1_1d

How can I fix this?
